docs/source/common/optimizers.rst

```diff
@@ -592,3 +593,21 @@ Here we configure gradient clipping differently for optimizer B.
             self.clip_gradients(
                 optimizer, gradient_clip_val=gradient_clip_val * 2, gradient_clip_algorithm=gradient_clip_algorithm
             )
+
+
+Total Stepping Batches
```

Comment:
@rohitgr7 I had a question on this name

Why not total optimization steps? People often get confused about the terminology in Lightning between steps and batches, and this property combines both of them. Is the concept of a batch even relevant here, since it's really related to model weight updates?

Reply:
It is resolved? Or you still need some clarification?

Reply:
i had missed the discussion in the PR which answered my questions